Marlin/pins.h

```diff
@@ -308,7 +308,12 @@
 #define _E3_PINS
 #define _E4_PINS
 
-#if EXTRUDERS > 1
+#if ENABLED (SINGLENOZZLE_SWITCHING_EXTRUDER) // needs only one stepper for 2 Extruder
+  #if EXTRUDERS > 2
```

Comment:
E2 / E3 pins are not used, cause only max two stepper E0 and E1 ...

In this section a E4 Stepper (Extruder 5 !? )  is defined, i think it shoud be only max E3 !?